theano/tensor/elemwise.py

```diff
@@ -134,6 +134,7 @@ class DimShuffle(Op):
 
     _f16_ok = True
     check_input = False
+    __props__ = ("input_broadcastable", "new_order", "inplace")
```

Comment:
I think this was already in the PR that I merged about DimShuffle. Can you rebase this PR to remove this duplicate changes in both PR?